### PR TITLE
Readme and documentation update

### PR DESCRIPTION
**Pull request by MatrixCrawler**
Did some cleaning up with the readme and documentation

- removed the obsolete html files. They are not needed in this repo as mkdocs generates it from the markdown definitions.
- Added some infomation reagrding windows binaries.

### CODE REVIEW

**Review by yermulnik — COMMENTED** [review]
These two GIFs don't show for me eve nwhen downloaded. Is it me or them?
* https://github.com/warrensbox/terraform-switcher/blob/readme-doc-update/www/docs/static/contribute/tfswitch-workspace.gif
* https://github.com/warrensbox/terraform-switcher/blob/readme-doc-update/www/docs/static/versiontf.gif

Plus the whitespacing comments below.
Otherwise LGTM.

File: README.md [hunk 1]
```diff
@@ -1,344 +1,42 @@
...
-<img style="text-allign:center" src="https://s3.us-east-2.amazonaws.com/kepler-images/warrensbox/tfswitch/smallerlogo.png" alt="drawing" width="120" height="130"/>
+# Terraform Switcher
 
-<!-- ![gopher](https://s3.us-east-2.amazonaws.com/kepler-images/warrensbox/tfswitch/logo.png =100x20) -->
+<img style="text-align:center" src="https://s3.us-east-2.amazonaws.com/kepler-images/warrensbox/tfswitch/smallerlogo.png" alt="drawing" width="120" height="130"/>
 
-The `tfswitch` command line tool lets you switch between different versions of [terraform](https://www.terraform.io/).
-If you do not have a particular version of terraform installed, `tfswitch` will download and verify the version you desire.
-The installation is minimal and easy.
+The `tfswitch` command line tool lets you switch between different versions of [terraform](https://www.terraform.io/).
```
Comment (crablab) [inline README.md:11]:
```suggestion
The `tfswitch` command line tool lets you switch between different versions of [Terraform](https://www.terraform.io/).  
```

File: README.md [hunk 2]
```diff
@@ -1,344 +1,42 @@
...
+# Terraform Switcher
 
-<!-- ![gopher](https://s3.us-east-2.amazonaws.com/kepler-images/warrensbox/tfswitch/logo.png =100x20) -->
+<img style="text-align:center" src="https://s3.us-east-2.amazonaws.com/kepler-images/warrensbox/tfswitch/smallerlogo.png" alt="drawing" width="120" height="130"/>
 
-The `tfswitch` command line tool lets you switch between different versions of [terraform](https://www.terraform.io/).
-If you do not have a particular version of terraform installed, `tfswitch` will download and verify the version you desire.
-The installation is minimal and easy.
+The `tfswitch` command line tool lets you switch between different versions of [terraform](https://www.terraform.io/).  
+If you do not have a particular version of terraform installed, `tfswitch` will download and verify the version you desire.
```
Comment (crablab) [inline README.md:12]:
```suggestion
If you do not have a particular version of Terraform installed, `tfswitch` will download and verify the version you desire.  
```

File: README.md [hunk 3]
```diff
@@ -1,344 +1,42 @@
...
-<!-- ![gopher](https://s3.us-east-2.amazonaws.com/kepler-images/warrensbox/tfswitch/logo.png =100x20) -->
+<img style="text-align:center" src="https://s3.us-east-2.amazonaws.com/kepler-images/warrensbox/tfswitch/smallerlogo.png" alt="drawing" width="120" height="130"/>
 
-The `tfswitch` command line tool lets you switch between different versions of [terraform](https://www.terraform.io/).
-If you do not have a particular version of terraform installed, `tfswitch` will download and verify the version you desire.
-The installation is minimal and easy.
+The `tfswitch` command line tool lets you switch between different versions of [terraform](https://www.terraform.io/).  
+If you do not have a particular version of terraform installed, `tfswitch` will download and verify the version you desire.  
+The installation is minimal and easy.  
 Once installed, simply select the version you require from the dropdown and start using terraform.
```
Comment (crablab) [inline README.md:14]:
```suggestion
Once installed, simply select the version you require from the dropdown and start using Terraform.
```

File: README.md [hunk 4]
```diff
@@ -1,344 +1,42 @@
...
-2. Ex: `tfswitch -d 1.2.3` or `tfswitch --default 1.2.3` installs version `1.2.3` when no other versions could be detected.
-[Also, see CICD example](#cicd)
-
-## Automation
-**Automatically switch with bash**
-
-Add the following to the end of your `~/.bashrc` file:
-(Use either `.tfswitchrc` or `.tfswitch.toml` or `.terraform-version`)
+## Documentation
+Click [here](https://tfswitch.warrensbox.com) for our extended documentation.
```
Comment (crablab) [inline README.md:17]: I don't think we should entirely remove the installation instructions! 

Maybe we can have a subheading for Installation with a link to the relevant install page here instead, including listing all the ways installation is supported? (eg. via Brew, build from source, go get etc)

File: www/docs/Installation.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+# Installation
+`tfswitch` is available for Windows, macOS and Linux based operating systems.
+
+## Windows
+Download and extract the Windows version of `tfswitch` that is compatible with your system.  
+We are building binaries for 386, amd64, arm6 and arm7 CPU structure.  
+See the [release page](https://github.com/warrensbox/terraform-switcher/releases/latest) for your download.
+
+## Homebrew
+Installation for macOS is the easiest with Homebrew. <a href="https://brew.sh/" target="_blank">If you do not have homebrew installed, click here</a>.
```
Comment (crablab) [inline www/docs/Installation.md:10]:
```suggestion
Installation for macOS is the easiest with Homebrew. If you do not have homebrew installed, [click here](https://brew.sh).
```

File: README.md [hunk 6]
```diff
@@ -1,344 +1,42 @@
...
-  cdir=$PWD;
-  if [ -e "$cdir/.tfswitchrc" ]; then
-    tfswitch
-  fi
-}
-alias cd='cdtfswitch'
-```
+## Quick Start
+### Dropdown Menu
+Just call `tfswitch` and select the desired terraform version via the dropdown menu.
```
Comment (crablab) [inline README.md:21]:
```suggestion
[Once installed](https://tfswitch.warrensbox.com/Installation/), execute `tfswitch` and select the desired Terraform version via the dropdown menu.
```

File: README.md [hunk 7]
```diff
@@ -1,344 +1,42 @@
...
+### Version on command line
+Call `tfswitch 1.7.0` to install terraform in the version 1.7.0. Replace version string with your desired version.
```
Comment (crablab) [inline README.md:23]:
```suggestion
Use `tfswitch 1.7.0` to install Terraform version 1.7.0. Replace the version number as required.
```

File: README.md (hunk 4 above)
Comment (yermulnik) [reply README.md:17]: +1

File: README.md [hunk 8]
```diff
@@ -1,344 +1,49 @@
...
-| 1 | .tfswitch.toml |
-| 2 | .tfswitchrc |
-| 3 | .terraform-version |
-| 4 | Environment variable |
-
-With 1 being the highest precedence and 4 the lowest   
-*(If you disagree with this order of precedence, please open an issue)*
-## How to contribute    
+## Installation
+`tfswitch` is available as a binary and on various package managers (eg. brew).
```
Comment (yermulnik) [inline README.md:19]:
```suggestion
`tfswitch` is available as a binary and on various package managers (eg. Homebrew). 
```

File: README.md [hunk 9]
```diff
@@ -1,344 +1,49 @@
...
-With 1 being the highest precedence and 4 the lowest   
-*(If you disagree with this order of precedence, please open an issue)*
-## How to contribute    
+## Installation
+`tfswitch` is available as a binary and on various package managers (eg. brew). 
+
+See [our installation documentation](https://tfswitch.warrensbox.com/Installation) for more details. 
+
+> [!IMPORTANT]
+> The version identifier of `tfswitch` has changed to align with the common `go` package versioning.
```
Comment (yermulnik) [inline README.md:24]: What are those trailing single spaces for? 🤔

Comment (crablab) [reply README.md:24]: There are many in this file - I just split the lines up. I'll remove them.

**Comment by yermulnik** [comment]
> I'll wait for @yermulnik to ✔️ too.

So are those GIFs "broken" for me only? https://github.com/warrensbox/terraform-switcher/pull/376#pullrequestreview-1998223248

**Comment by crablab** [comment]
> So are those GIFs "broken" for me only?

Ah apologies - no, they are also broken for me.

**Comment by crablab** [comment]
The issue seems to be git has converted line breaks incorrectly. [See here](https://stackoverflow.com/questions/52744233/how-to-add-to-git-binary-file-which-has-both-crlf-and-lf-symbols-inside). 

I've tried a few things but unfortunately cannot get git to accept the unconverted file, and `dos2unix` is corrupting the binary too.

**Comment by yermulnik** [comment]
@crablab Send me files via email and I'll see if I can add them correctly (I'm on Linux).

**Comment by crablab** [comment]
So am I @yermulnik 🙂

I'll send them to you today.

**Comment by MatrixCrawler** [comment]
I think it might be becaus i forgot to add the *.gif as binary into the gitattributes so git changed the CLRF to LF
The files on my system should still work. I think i can fix it.

**Comment by MatrixCrawler** [comment]
@crablab @yermulnik Fixed the gif files.

**Review by yermulnik — APPROVED** [review]

**Pull request merged**